VERSION

```diff
@@ -106,17 +106,20 @@ date="Unreleased developer copy"
 # All changes in these version numbers are dictated by the PMIx
 # release managers (not individual developers).  Notes:
 
-# 1. Since these version numbers are associated with *releases*, the
-# version numbers maintained on the PMIx Github trunk (and developer
-# branches) is always 0:0:0 for all libraries.
+# 1. Since the PMIx master branch is maintained to always be compatible
+#    with all prior releases (i.e., we never remove an interface), the
+#    libtool 'c' and 'a' values are maintained equal to the highest values
+#    found on any given release branch. This ensure that the .so value
+#    remains the same as on the release branches, thereby indicating
+#    compatibility.
```

Comment:
You actually need to maintain all three across the board.  The 4.2 branch also has the version as 9:0:7, so the code is right, but the comment is wrong (I believe).

Reply:
oh, yeah - I changed the code and forgot to update the comment. I'll fix that - thanks for catching it!